app/controllers/deploys_controller.rb

```diff
@@ -71,7 +71,8 @@ def search
       stages = Stage
       stages = stages.where(project: projects) if projects
       if params[:production].present?
-        production = ActiveRecord::Type::Boolean.new.type_cast_from_user(params[:production])
+        production = params[:production]
+        production = production.nil? || !ActiveModel::Type::Boolean::FALSE_VALUES.include?(production)
```

Comment:
`ActiveRecord::Type::Boolean.new.type_cast_from_user(nil) -> nil` ... so no ...
so changing to !nil && !include